README.md

```diff
@@ -8,25 +8,29 @@ The list of providers is published in the form of a statically hosted OPTiMaDe I
 
 If you specifically seek the current list of providers for the latest version of the OPTiMaDe specification, you can access it at this URL:
 
-- [https://www.optimade.org/providers/links/](https://www.optimade.org/providers/links/)
+- [https://www.optimade.org/providers/providers.json](https://www.optimade.org/providers/providers.json)
 
 If you seek the current list of providers for any older version of the OPTiMaDe specification, you can access it using this URL:
 
-- [https://www.optimade.org/providers/*\<version\>*/links/](https://www.optimade.org/providers/<version>/links/)
+- [https://www.optimade.org/providers/*\<version\>*/links](https://www.optimade.org/providers/<version>/links)
 
```

Comment:
If '.../providers/links/' was renamed to `.../providers/providers.json`, wouldn't it be more consistent to rename also the version specific URIs, to `.../providers/<version>/providers.json`?

Reply:
See general comment elsewhere. In short, these are named as such because they represent endpoints in the OPTiMaDe specification.